scripts/migrate-featured-to-urls.py

```
@@ -0,0 +1,128 @@
+        )
+        return json.loads(out) if out else []
+    return run_sql(target, query)
+
+
+def update_row(target: str, username: str, urls: list[str]):
+    array_sql = "array[" + ",".join("'" + u.replace("'", "''") + "'" for u in urls) + "]::text[]"
+    if not urls:
+        array_sql = "array[]::text[]"
+    query = (
+        f"update users set featured_github_prs = {array_sql} "
+        f"where github_username = '{username}'"
```

Ruff (0.16.1)

[error] Possible SQL injection vector through string-based query construction

(S608)